### ISSUE TITLE
A11y: Digital Products - Price display card and shopping card seems broken

### ISSUE THREAD

**Issue by shopwareBot** [shopware/platform#5099]
After new digital product epic is added to the trunk, when we use custom products with digital products, price display card on storefront seems broken. 

It is also reproducible for 6.4.

*Steps to reproduce:*

* create CUS template
* create digital product 
* assign CUS template to the digital product
* go to storefront and check if it is product detail page 

*Actual Result:*

* Product display card seems broken and then if you configure sth and add product to the card, offcanvas will give also 500 error.

*Expected Result*

* Product price display card and shopping card should be visible

**Comment by shopwareBot** [comment]
[public] Shopware automated response: This issue is linked to the internal issue: **CUS-584**



[created from CUS-584, comment 503003]

**Comment by shopwareBot** [comment]
[public]

 > Status Change: **In Progress**

 Work on this issue has started. It is now being implemented.

 

[created from CUS-584, comment 511817]

**Comment by shopwareBot** [comment]
[public]

 > Status Change: **In Review**

 Work on this ticket is complete and is now pending review.



[created from CUS-584, comment 512554]
